theano/compile/profiling.py

```diff
@@ -109,6 +109,12 @@ def _atexit_print_fn():
                     n_apply_to_print=config.profiling.n_apply)
 
 
+_profiler_printers = []
+
+def register_profiler_printer(fct):
+    profiler_printers.append(fct)
```

Comment:
There is a mistake here that make travis fail: `profiler_printers` -> `_profiler_printers`